packages/api/src/app/api/protocol-params.ts

```diff
@@ -0,0 +1,103 @@
+import {z} from 'zod'
+import {createTypeGuardFromSchema, fetcher, Fetcher} from '@yoroi/common'
+import {App} from '@yoroi/types'
+
+export const getProtocolParams =
+  (baseUrl: string, request: Fetcher = fetcher) =>
+  async (): Promise<any> => {
+    console.log('[baseUrl]', baseUrl)
+    return request<App.FrontendProtocolParamsResponse>({
+      // url: `${baseUrl}/protocolparameters`, // only on preprod
+      url: 'https://dev-yoroi-backend-zero-preprod.emurgornd.com/protocolparameters',
+      data: undefined,
+      method: 'GET',
+      headers: {'Content-Type': 'application/json'},
+    }).then((response) => {
+      const parsedResponse = parseFrontendProtocolParamsResponse(response)
+
+      if (!parsedResponse)
+        return Promise.reject(new Error('Invalid protocol params response'))
+      return Promise.resolve(parsedResponse)
+    })
+  }
+
+const AppFrontendProtocolParamsTierSchema = z.object({
+  epoch: z.number(),
+  min_fee_a: z.number(),
+  min_fee_b: z.number(),
+  max_block_size: z.number(),
+  max_tx_size: z.number(),
+  max_block_header_size: z.number(),
+  key_deposit: z.string(),
+  pool_deposit: z.string(),
+  min_utxo: z.string(),
+  min_pool_cost: z.string(),
+  price_mem: z.number(),
+  price_step: z.number(),
+  max_tx_ex_mem: z.string(),
+  max_tx_ex_steps: z.string(),
+  max_block_ex_mem: z.string(),
+  max_block_ex_steps: z.string(),
+  max_val_size: z.string(),
+  collateral_percent: z.number(),
+  max_collateral_inputs: z.number(),
+  coins_per_utxo_size: z.string(),
+  coins_per_utxo_word: z.string(),
+})
+
+const AppFrontendFeesResponseSchema = AppFrontendProtocolParamsTierSchema
+
+export const isFrontendProtocolParamsResponse = createTypeGuardFromSchema(
+  AppFrontendFeesResponseSchema,
+)
+
+export const parseFrontendProtocolParamsResponse = (
+  data: App.FrontendProtocolParamsResponse,
+): App.FrontendProtocolParamsResponse | undefined => {
+  const filteredObject = pick(data, [
+    'epoch',
```

Comment:
have filtered some of the data as the endpoint response is quite big. I'm thinking that we can return only the constants that we need from the api package